lesson24/script.js

```diff
@@ -0,0 +1,92 @@
+let numberFirst = prompt('number1');
+let numberSecond = prompt('number2');
+
+function checking(value) {
+    if (value === 0 || value) {
```

Comment:
I know that it's from lesson example, but now I see that this checking isn't perfect because string `'abc'` or array `[]` also will pass your `if`

so better change to:
`if (typeof value === 'number' && !Number.isNaN(value)) `